buildchain/buildchain/salt_tree.py

```diff
@@ -59,6 +59,46 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
         yield from file_tree.execution_plan
 
 
+class ConstantContainerRegistry(targets.AtomicTarget):
+    """Generate a nginx configuration to serve constants container registry."""
```

Comment:
What is a `constants container registry`?

Reply:
It's like `static container registry` ...

Reply:
Btw if you have better naming I'm ok to change this

Reply:
`static container registry` means, I guess, that's it's a read-only registry whose list of images is generated at build time (hence the `static`, opposite to `dynamic` where you could push images on the fly).

Reply:
Yeah, please don't use the term 'constant' here, that's not the right vocabulary.

Reply:
Done